qpc_tools/cli/ansible/install/roles/cli/tasks/load_rpm_online.yml

```diff
@@ -46,8 +46,14 @@
   when:
     - cli_version is not defined
 
+- name: Set the Quipucords CLI RPM package name
+  set_fact:
+    cli_package_name: "{{ cli_github_release_url }}/qpc.el{{ ansible_distribution_major_version }}.noarch.rpm"
+  when:
+    - cli_package_name == ""
+
 - name: Install the Quipucords CLI
   yum:
-    name: '{{ cli_github_release_url }}/{{ qpc_package_name }}'
+    name: '{{ cli_package_name }}'
```

Comment:
Revert this and replace this whole file downstream with a small role that does a simple yum install.